sccameraview/src/main/java/com/example/sccameraview/Camera1View.java

```diff
@@ -68,6 +64,17 @@ void openCamera() {
     }
 
     @Override
+    public void takePicture() {
+        camera.takePicture(null, null, null, new Camera.PictureCallback() {
```

Comment:
Size, flash, and autofocus are set when the camera is opened, the camera1 API callbacks work with those values.